prowler/providers/aws/services/drs/drs_jobs_exist/drs_jobs_exist.py

```diff
@@ -0,0 +1,23 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.drs.drs_client import drs_client
+
+
+class drs_jobs_exist(Check):
+    def execute(self):
+        findings = []
+        report = Check_Report_AWS(self.metadata())
+        report.status = "FAIL"
+        report.status_extended = "No DRS jobs exist."
+        report.resource_id = "DRS"
+        report.region = drs_client.region
+        report.resource_tags = []
+        report.resource_arn = ""
+        if drs_client.drs_jobs:
```

Comment:
Is this a global service? If not we need to create a dummy object per instance to set the status of DRS and then check if there are jobs or not. What do you think?